03-Bash/fix_owner/fix_owner.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+file="/etc/passwd"
+IFS=$'\n'
+for user_line in $(cat $file)
+do
+    declare -i user_id
+    user_id=$(echo $user_line | cut -d ':' -f3)
+    if [ $user_id -lt 1000 ]; then
+        continue
+    fi
+    user_name=$(echo $user_line | cut -d ':' -f1)
+    user_home=$(echo $user_line | cut -d ':' -f6)
+    user_group=$(echo $user_line | cut -d ':' -f4)
+
+    # echo "$user_name=$user_id:$user_group-$user_home"
+
+    find $user_home -maxdepth 1 > homeFileList
```

Comment:
Consider using unique temporary file.

Reply:
Should this file be stored in /tmp?

Reply:
Yes, it is usual place to temporary files.

Reply:
Also mktemp utility can be used.

Reply:
Used mktemp